core/math/quat.cpp

```diff
@@ -232,6 +232,8 @@ Quat::operator String() const {
 }
 
 void Quat::set_axis_angle(const Vector3 &axis, const real_t &angle) {
+	ERR_EXPLAIN("set_axis_angle is deprecated. Use the Quat(Vector3 axis, float angle) constructor instead.");
```

Comment:
You could remove the duplicated code below by a call to the constructor I guess.